src/game/audio/sound_effects.rs

```diff
@@ -0,0 +1,100 @@
+//! Functions and types for playing sound effects in the game.
+//! The main thing of interest here is the extension trait `SoundEffectCommands`
+//! and its `play_sound_effect` method.
+//!
+//! This pattern is adapted from the [Bevy example for sound effects](https://github.com/bevyengine/bevy/pull/14554).
+
+use bevy::{ecs::world::Command, prelude::*};
+use rand::{distributions::Uniform, Rng as _};
+
+use crate::game::assets::SoundEffectHandles;
+
+pub(super) fn plugin(_app: &mut App) {
+    // No setup required for this plugin.
+    // It's still good to have a function here so that you can add some setup later if needed.
```

Comment:
I don't agree with this personally.

Reply:
I think not adding a plugin when nearly all other files have a plugin is potentially more confusing to a user than the nuance is worth. I personally would add a `plugin` here because it is very realistic that a `register_type` or `observe` might one day get in here, and I don't want to be annoyed that I have to go back and wire the plugin up. One plugin per file is a good rule of thumb imo.

Reply:
Not strongly enough to block this PR. Can always discuss this later.